src/components/pager/pager.component.tsx

```diff
@@ -20,7 +20,7 @@ type PageSizeOption = {
 
 export interface PagerProps {
   /** Function called when pager changes (PageSize, Current Page) */
```

Comment:
**suggestion:** You will also need to update this comment to reflect the changes you've made to the ordering of the parameters.

Reply:
thanks for pointing that out. Done.